src/ci/docker/run.sh

```diff
@@ -219,6 +219,10 @@ else
   command="/checkout/src/ci/run.sh"
 fi
 
+# Get some needed information for $BASE_COMMIT
+git fetch "https://github.com/$GITHUB_REPOSITORY" "$GITHUB_BASE_REF"
+BASE_COMMIT="$(git merge-base FETCH_HEAD HEAD)"
```

Comment:
I think this is going to cause problems when trying to run these scripts locally -- GITHUB_BASE_REF and such are not well defined really. I think blocking this on `if [ "$CI" != "" ]; then` (like we do above in this file) and making sure that the validate-error-codes script only runs if BASE_COMMIT is defined.